frontend/packages/client/src/App.js

```diff
@@ -34,7 +33,7 @@ function App() {
             </Router>
           </NotificationModalProvider>
         </Web3Provider>
-        {isLocalDevMode && <ReactQueryDevtools />}
+        <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
if NODE_ENV is not development then this component doesn't get added to the bundle and does not apear on the app.